app.js

```diff
@@ -1,164 +1,524 @@
+          // Parse additional args
+          logDump(bot, channelID, flags);
+          break;
+        case 'env': // Return environment
+          bot.sendMessage({
+            to: channelID,
+            message: "```JSON\nEnvironment:\n" +
+              `Process architecture: ${process.env.PROCESSOR_ARCHITECTURE}\n` +
+              `Default channel ID: ${weeabooChannelId}\n` +
+              `Current channel ID: ${channelID}\n` +
+              `Bot ID: ${bot.id}\n` +
+              `Followed: ${bot.learningTargetId} == ` +
+              `"${bot.users[bot.learningTargetId].username}"\n` +
+              "```"
+          })
+          break;
+        case 'help': // Get command list
+          formattedList = cmd_list.sort().map(s => '`!' + s + '` ').join(" ");
+          bot.sendMessage({
+            to: channelID,
+            message: `Here's the list of commands, although I'd rather sleep ${maxsnuzyenEmoji}: \n${formattedList}`
+          })
+          break;
+        case 'set': // Set properties
+          validateAndSet(bot, flags);
+          break;
+        case 'msgs': // Upload messages list
+          bot.uploadFile({
+            to: channelID,
+            file: messageLogName,
+            message: "Message list uploaded."
+          })
+          break;
+        case 'who': // Return who is on channel
+          fs.writeFile(`${userLogName}`, JSON.stringify(bot.users), (err, data) => {
+            if (err)
+              logger.error("Error writing users file: " + err.message);
+            bot.uploadFile({
                 to: channelID,
-                message: "<@" + userID + "> Wait, who's that? Tag 'em"
-              });
-            }
+                file: userLogName,
+                message: "Users file uploaded."
+            })
+          })
+          break;
+        case 'goodbot': // Yay
+          bot.addReaction({
+            channelID: channelID,
+            messageID: evt.d.id,
+            reaction: "POGGERS:427999200337461248"
+          }, (err) => {
+            if (err) {console.log(err); logger.error(`Reaction error: ${err.message}`)}
+          });
+          break;
+        case 'badbot': // Aww
+          bot.addReaction({
+            channelID: channelID,
+            messageID: evt.d.id,
+            reaction: "maxsnuzyen:489283891807518720"
+          }, (err) => {
+            if (err) {console.log(err); logger.error(`Reaction error: ${err.message}`)}
+          });
+          break;
+        case 'yikes':
+          bot.addReaction({
+            channelID: channelID,
+            messageID: evt.d.id,
+            reaction: "monkaS:506646811688173579"
+          }, (err) => {
+            if (err) {console.log(err); logger.error(`Reaction error: ${err.message}`)}
+          });
+          break;
+        case 'maximback':
+          interruptTimer(bot);
+          break;
+        case 'db': // Interactive debugger
+          if (flags.off && bot.isInteractive){
```

Comment:
Why are there flags for interactive mode tho?
Shouldn't it just be !db?
Then you can toggle with 
```
if (bot.isInteractive) {
         ....
} else {
    ...
}
```